apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -166,14 +166,16 @@ const SlotPicker = ({
 
     // Etc/GMT is not actually a timeZone, so handle this select option explicitly to prevent a hard crash.
     if (timeZone === "Etc/GMT") {
-      setBrowsingDate(dayjs.utc(month).startOf("month"));
+      setBrowsingDate(dayjs.utc(month).set("date", 1).set("hour", 0).set("minute", 0).set("second", 0));
       if (date) {
         setSelectedDate(dayjs.utc(date));
       }
     } else {
-      setBrowsingDate(dayjs(month).tz(timeZone, true).startOf("month"));
+      setBrowsingDate(
```

Comment:
`startOf` is based on server/browser time, pretty much making it useless.

Reply:
Same here. Let's add some comments to avoid shouting ourselves in the foot